libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -817,6 +817,7 @@ def bind_tools(
         tool_choice: Optional[
             Union[dict, str, Literal["auto", "none", "required", "any"], bool]
         ] = None,
+        parallel_tool_calls: Optional[bool] = None,
```

Comment:
since we're not doing anything with this param (just plumbing through kwargs) we actually don't even need to add it to function signature